exercises/ex4/README.md

```diff
@@ -12,32 +12,13 @@ Remember, we want to add a goal and add tasks to the goal when a user registers
 
 So let's enhance the `GoalServiceHandler` to implement these interactions with the SuccessFactors API.
 
-## 4.1 Create a Remote Service Object
-
-Similarly to the OpenAPI consumption discussed in exercise 3 we need to define a service object that we will use to run our OData queries.
-
-- [ ] 🔨 **Add the following field to the `GoalServiceHandler` class:**
-
-    ```java
-    @Autowired
-    @Qualifier(cds.gen.goal.Goal_.CDS_NAME)
-    private CqnService goalService;
-    ```
-
-// the tip below seems to be out of place
-> **Tip:** Treat the package names with caution. Depending on how you name your CDS entities and services as well as how remote services are named there may be naming clashes. 
-> 
-> In our case the generated API class for the SuccessFactors service is similar to the generated entity class `cds.gen.goalservice.Goal_` of our `GoalServcie`. So the fully qualified class name is shown here.
-> 
-> Feel free to rename the `cds.gen.goalservice.Goal_` entity later as an optional exercise.
```

Comment:
I removed this paragraph as I added the corresponding code (commented) to the class already. Without the code added, the application cannot be compiled at an earlier exercise.

Reply:
No, should be everything I think.

Reply:
Done as part of https://github.com/SAP-samples/teched2023-AD266/pull/17/commits/866204cbbcb3e3296b38c5db2f5cef7d8c7d796b